task/eaas-provision-space/0.1/eaas-provision-space.yaml

```diff
@@ -59,20 +59,36 @@ spec:
         esac
 
         cat <<EOF > space_request.yaml
-        apiVersion: toolchain.dev.openshift.com/v1alpha1
-        kind: SpaceRequest
+        apiVersion: apiextensions.crossplane.io/v1
+        kind: CompositeResourceDefinition
```

Comment:
This task shouldn't create an XRD. It needs to create a namespace claim defined by the XRD which is already deployed. Something like this: https://github.com/konflux-ci/crossplane-control-plane/blob/main/examples/xnamespace/claim.yaml.

Please provide some evidence that these changes have been tested (in staging or prod) because it's pretty clear this would fail (the pipeline runner SA cannot create XRDs).